backend/routeHandler.js

```diff
@@ -74,6 +74,50 @@ module.exports = function configureRoutes(options) {
     res.send(JSON.stringify(query));
   });
 
+  options.router.post('/getQueryByName', async (req, res) => {
+    try {
+    // Checking templates
+      const getQueryParams = {
+        sourceKey: req.body.sourceKey,
+        type: "template"
+      };
+
+      let response = (await options.getRestClient(req).graphQuery.getQueries(getQueryParams));
+
+      let queries = response.body.filter((q) => {
+        return q.name.toLowerCase() === req.body.name;
```

Comment:
@andre91a I would expect the need to match with req.body.name.toLowerCase(), otherwise you'll miss some match.

Reply:
@fantandrea Apparently the backend always receives the lowerCased `req.body.name` parameter: I can lowerCase it but it would be redundant

Reply:
btw, I've decided to compare the lowerCased values because the LKE show capitalised names for the queries (that are not always matching the real name of the query): in the plugin, we lose some specificity in query selection but make it easier to use

Reply:
I've double checked: `req.body.name` is lowerCased only when there are no URL encoded characters:

`?queryName=myTest` => `mytest`
`?queryName=my%20Test` => `my Test`

Fixing it